### PR TITLE
feat: add the money talk notes

### PR DESCRIPTION
**Pull request by dariagrudzien**
Post preview available at https://deploy-preview-40--infallible-sinoussi-4e1719.netlify.app/

Scheduled for release on April 12th.

### CODE REVIEW

File: content/blog/the-money-talk-meetup.md [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+current rates for your type of role in the market you work in. Do solid
+research (we'll talk about it in the next step) and treat this
+discussion as any other merit-based professional conversation that you
+prepare for.
+
+### Accept The Process
+
+
+There's an important point you need to acknowledge---negotiating salary
+is part of the process of finding a new job or raising to the next level
```
Comment (anezkamll) [inline content/blog/the-money-talk-meetup.md:69]:
```suggestion
is a part of the process of finding a new job or raising to the next level
```

File: content/blog/the-money-talk-meetup.md [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+in your current organization. There is an asymmetry of information in
+the professional context because your employer knows its own budget and
+salaries of everyone employed at the company, but you usually don't. In
+that situation the company is benefiting from the lack of transparency
```
Comment (anezkamll) [inline content/blog/the-money-talk-meetup.md:73]:
```suggestion
that situation, the company is benefiting from the lack of transparency
```

File: content/blog/the-money-talk-meetup.md [hunk 3]
```diff
@@ -0,0 +1,261 @@
...
+and if they wouldn't want salary negotiations to be a part of the
+process, they'd introduce some level to transparency (e.g. by providing
+official salary ranges.)
+
+## Do The Research
+
+Figuring out the value of your work compared to the current market
+situation is a necessary starting point. Here's how to make sure you get
+most correct data:
```
Comment (anezkamll) [inline content/blog/the-money-talk-meetup.md:82]:
```suggestion
the most correct data:
```

File: content/blog/the-money-talk-meetup.md [hunk 4]
```diff
@@ -0,0 +1,261 @@
...
+Bear in mind that the process of negotiation may affect the quality of
+the relationship with your future managers. Literature describes a
+phenomenon known as ["the social cost of
+negotiating"](https://women.govt.nz/nacew/information/develop-your-workplace-negotiation-skills/why-negotiation-issue-women/social-costs)---it's
+important to know how to phrase the discussion to appear [less
+combative](https://hbr.org/2014/06/why-women-dont-negotiate-their-job-offers).
+Make sure to watch [Margaret Neale's (Stanford Graduate School of
+Business)
+video](https://www.youtube.com/watch?v=MXFpOWDAhvM) about
+framing negotiation as a problem solving exercise with the focus on
```
Comment (anezkamll) [inline content/blog/the-money-talk-meetup.md:146]:
```suggestion
framing negotiation as a problem-solving exercise with the focus on
```

File: content/blog/the-money-talk-meetup.md [hunk 5]
```diff
@@ -0,0 +1,261 @@
...
+One thing to keep in mind---don't let this issue get in the way of
+negotiating your salary. US research shows that giving up on negotiation
+can cost Americans [1 million dollars over the span of their
+career](https://www.businessinsider.com/not-negotiating-costs-workers-1-million-2013-9).
+Now, a million dollars is a good motivation to overcome the negotiation
+stress, isn't it?
+
+### When Does It Make Sense To Talk About Numbers
+
+The answer to that question is unfortunately "It depends." While a bulk
```
Comment (anezkamll) [inline content/blog/the-money-talk-meetup.md:158]:
```suggestion
The answer to that question is, unfortunately, "It depends." While a bulk
```

File: content/blog/the-money-talk-meetup.md [hunk 6]
```diff
@@ -0,0 +1,261 @@
...
+impressive than your full skillset, then it really makes sense to leave
+this conversation for the end. Your strong performance in the interview
+will allow you to present yourself as a solid candidate.
+
+It's fair to explain that you'd like to fully understand the range of
+responsibilities and the details of the job before estimating your
+salary for that particular role. If the person asking for the financial
+expectations keeps pushing, you can counter by asking "Are you able to
+share the salary ranges that you have for that position with me?" At
+that point most people agree to discuss the numbers after the interview
```
Comment (anezkamll) [inline content/blog/the-money-talk-meetup.md:176]:
```suggestion
that point, most people agree to discuss the numbers after the interview
```

File: content/blog/the-money-talk-meetup.md [hunk 7]
```diff
@@ -0,0 +1,261 @@
...
+even a bit more. It seems there's a lower relationship penalty for
+setting a more aspirational upper bound as long as the lower bound is
+reasonable.
+
+### Make Sure You Always Have An Alternative
+
+In addition to having solid research to back up your ask, it's necessary
+to evaluate your alternative options to establish what's known as
+[BATNA](https://www.storyboardthat.com/articles/b/best-alternative-to-negotiated-agreement)---Best
+Alternative To Negotiated Agreement. To put it simply you need to know
```
Comment (anezkamll) [inline content/blog/the-money-talk-meetup.md:200]:
```suggestion
Alternative To Negotiated Agreement. To put it simply, you need to know
```

File: content/blog/the-money-talk-meetup.md [hunk 8]
```diff
@@ -0,0 +1,261 @@
...
+what are your options if the negotiation will fail. Can you simply stay
```
Comment (anezkamll) [inline content/blog/the-money-talk-meetup.md:201]:
```suggestion
what are your options if the negotiation fails. Can you simply stay
```

File: content/blog/the-money-talk-meetup.md [hunk 9]
```diff
@@ -0,0 +1,261 @@
...
+
+It may happen that when you hand in your resignation your current
+employer will come back to you with a counteroffer---a promise to raise
+your salary to keep you in the team. Make sure to [think it
+through](https://www.apollotechnical.com/reasons-to-not-accept-a-counteroffe)
+before accepting the counter. While everyone's circumstances differ, the
+potential relief caused by the thought that you won't have to go through
+the stress of changing workplaces frequently isn't worth it. The
+original issues that made you decide to job hunt will not be
+autoresolved, and there may be some negative feelings on both sides.
```
Comment (anezkamll) [inline content/blog/the-money-talk-meetup.md:236]:
```suggestion
auto resolved, and there may be some negative feelings on both sides.
```

File: content/blog/the-money-talk-meetup.md [hunk 10]
```diff
@@ -0,0 +1,261 @@
...
+
+### Finding A Job That Values Your Full Skillset
+
+
+If you receive a job offer with a salary far below what you feel is
+fair, and the employer isn't willing to meet your numbers, it's
+perfectly fine to walk away from the offer. It may happen that your
+skillset is not the right fit for their role, or they are looking for
+just a fraction of what you can bring to the new team. In that case you
```
Comment (anezkamll) [inline content/blog/the-money-talk-meetup.md:245]:
```suggestion
just a fraction of what you can bring to the new team. In that case, you
```

**Review by anezkamll — APPROVED** [review]
Really well-written blog post @dariagrudzien I like it a lot :)
Just a few minor suggestions from my side, not necessary to accept them.

**Comment by dariagrudzien** [comment]
@anezkamll Thank you for the review, I appreciate it a lot! If you don't mind, I'll always ask for a review from now on when posting something and give you 2-3 days to get to it if you can.
I always feel guilty when I merge without a review >.>

**Comment by anezkamll** [comment]
> If you don't mind, I'll always ask for a review from now on when posting something and give you 2-3 days to get to it if you can.
> I always feel guilty when I merge without a review >.>
> 

Sure, happy to help this way :)

**Pull request merged**